apps/avifenc.c

```diff
@@ -398,6 +399,68 @@ static avifBool avifImageSplitGrid(const avifImage * gridSplitImage, uint32_t gr
     return AVIF_TRUE;
 }
 
+static int countLeadingZeros(uint32_t n)
```

Comment:
What about
```cpp
static int countLeadingZeros(uint32_t n)
{
    int count = 32;
    while (n != 0) {
        --count;
        n >>= 1;
    }
    return count;
}
```

Reply:
Done. Thanks.

The original code was copied from libgav1, which also has optimized code (based on compiler intrinsics) for common CPU architectures. In this use case, `n` is likely to be small, so your code is faster.